README.md

```diff
@@ -1,3 +1,5 @@
+![](https://github.com/kolorobot/spring-boot-junit5/workflows/tests/badge.svg)
```

Comment:
```suggestion
![](https://github.com/xmslyz/LuhnAlgorithm/workflows/tests/badge.svg)
```
To powinien być URL Twojego projektu. Badge (ten konkretny) informormuje, że workflow `tests` przechodzi na branchu `master` (albo nie przechodzi). Pokazuje stan najnowszego workflow w projektcie. Ten badge ma służyć pochwaleniu się, że tak dobrze piszesz kod, że testy do niego przechodzą. To sugeruje, że dbasz o jakość kodu i nieco podnosi wiarygodność projektu.  
więcej znajdziesz w [dokumentacji](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge).